app/controllers/forest_liana/smart_actions_controller.rb

```diff
@@ -0,0 +1,64 @@
+module ForestLiana
+  class SmartActionsController < ForestLiana::ApplicationController
+    if Rails::VERSION::MAJOR < 4
+      before_filter :check_permission_for_smart_route
+    else
+      before_action :check_permission_for_smart_route
+    end
+
+    private
+
+    def get_smart_action_request
```

Comment:
[Readability] I changed the name to `get_smart_action_request` because this is what we are getting actually. The model represented here is a `smart_action_request` from the UI.